docroot/modules/custom/foia_upload_xml/src/FailedMigrationHandler/SectionMissing.php

```diff
@@ -42,12 +42,13 @@ public function handle() {
    *   [
    *     'name' => 'component_xiic',
    *     'label' => 'Internal index of the agency component',
-   *     'selector' => 'foia:OldestPendingConsultationSection/foia:OldestPendingItems/@s:id',
+   *     'selector' =>
+   *     'foia:OldestPendingConsultationSection/foia:OldestPendingItems/@s:id',
    *   ]
    *
-   * @see source.fields in migrate_plus.migration.foia_agency_report.yml.
+   * @see fields
```

Comment:
It seems that you've replaced this with a much less useful comment. Can you fix this in a way that keeps the original information?